.gitignore

```diff
@@ -12,4 +12,6 @@ logs
 results
 
 node_modules
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+.idea/
```

Comment:
for what it's worth, i find it helpful to create a global .gitignore, rather than adding my IDE-specific ignores to every project I contribute to
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore